pkg/istio-agent/agent.go

```diff
@@ -320,6 +321,7 @@ func (a *Agent) initializeEnvoyAgent(ctx context.Context) error {
 				errCh:       a.dynamicBootstrapWaitCh,
 				envoyUpdate: envoyProxy.UpdateConfig,
 			}
+			a.xdsProxy.handlers[v3.BootstrapType] = bsStream.bootStrapHandler
```

Comment:
Added a specific handler as PCDS and NDS, so it can depend on the existing mechanism to ACK/NCK